src/components/AddItemForm.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import { withFirestore } from 'react-firestore';
+import { FirestoreCollection, withFirestore } from 'react-firestore';
 import '../CSS/AddItemForm.css';
 
 class AddItemForm extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       name: '',
+      name_normalized: '',
       next_purchase: 14,
       user_token: this.props.userToken,
```

Comment:
Probably don't need to keep this in state. We can always use `this.props.userToken` in the `onSubmit`